generator/operation.go

```diff
@@ -784,25 +796,34 @@ func (b *codeGenOpBuilder) MakeParameterItem(receiver, paramName, indexVar, path
 	res.Name = paramName
 	res.Path = path
 	res.Location = location
-	res.ValueExpression = swag.ToVarName(valueExpression)
+	res.ValueExpression = golang.MangleVarName(swag.ToGoName(valueExpression))
```

Comment:
why is there a need to change this?

Reply:
good catch. The ToVarName() is sufficient and issue #863 fix with items remains valid.